avocado/plugins/yaml_to_mux.py

```diff
@@ -262,8 +268,12 @@ def run(self, args):
             try:
                 args.mux.data_merge(create_from_yaml(multiplex_files, debug))
             except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+                if getattr(args, "mux_ignore_missing", 'off') == 'on':
+                    msg = 'Cannot access the multiplex file(s) in this host.'
```

Comment:
Well, taking to @adereis few moments ago we agreed in drop this option, keep the yaml_to_mux as is currently before this pr and document as a known issue that right now using remote + multiplex will require that the multiplex files are available both locally and remotely in the same path. The way out is to implement the plugins execution order to handle the local multiplex in the remote plugin, which must be engaged before the yaml_to_mux.